Controller/ResourceController.php

```diff
@@ -123,17 +123,38 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
         $page = (int) $request->get('page', 1);
-
+        $order = [];
         $filters = [];
-        foreach ($resource->getFilters() as $resourceFilter) {
+        $resourceFilters = $resource->getFilters();
+
+        foreach ($resourceFilters as $resourceFilter) {
             if (null !== $value = $request->get($resourceFilter['name'])) {
                 $resourceFilter['value'] = $value;
                 $filters[] = $resourceFilter;
             }
         }
 
+        // Add order filters
+        $requestOrderFilters = $request->get('order');
+        foreach ($requestOrderFilters as $key => $value) {
+            // @TODO change of whitelist, currently is the one of the where filter
+            foreach ($resourceFilters as $resourceFilter) {
+                if ($resourceFilter['name'] === $key
+                    && ('asc' === strtolower($value) || 'desc' === strtolower($value))) {
+                    $order[$key] = $value;
+                }
+            }
+        }
+
+        // If no order found take the default
+        if (0 === count($order)) {
```

Comment:
I prefer the old behavior because if the order wasn't explicitly provided, it default to the natural order of the DBMS or the default order defined in the Doctrine mapping.